content/docs/reference/autocert/autocert-ca.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: autocert-ca
+title: Autocert CA
+description: |
+  Autocert CA is the directory URL of the ACME CA to use when requesting certificates.
+keywords:
+  - reference
+  - Autocert CA
+pagination_prev: null
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Autocert CA
+
+## Summary
+
+**Autocert CA** is the directory URL of the ACME CA to use when requesting certificates.
+
+:::tip
+
+If set, Autocert CA will override the [**Autocert Use Staging**](/docs/reference/autocert/autocert-use-staging) setting.
+
+:::
+
+## How to configure
+
+| **Type** | **Usage**    |
+| :------- | :----------- |
+| `string` | **optional** |
+
+<Tabs>
+<TabItem value="Core" label="Core">
+
+| **Config file keys** | **Environment variables** |
+| :------------------- | :------------------------ |
+| `autocert_ca`        | `AUTOCERT_CA`             |
+
+</TabItem>
+<TabItem value="Enterprise" label="Enterprise">
+
+| **Config file keys** | **Environment variables** |
+| :------------------- | :------------------------ |
+| `autocert_ca`        | `AUTOCERT_CA`             |
```

Comment:
Looks wrong

Reply:
@desimone so this is what happens when you run `yarn format` (prettier) -- it adds dashes so that each row/column has the same amount of characters.